src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.ts

```diff
@@ -37,12 +37,15 @@ import { AppConfig } from 'app/app.config';
 })
 export class RetrospectTaskModalComponent implements OnDestroy, AfterViewChecked {
     sprintMembers: any;
+    retrospectiveID: any;
     taskDetails: any;
     selectedMemberID: number;
     gridOptions: GridOptions;
     enableRefresh: boolean;
     autoRefreshCurrentState: boolean;
     issueDescriptionHTML: string;
+    // To ignore column states updation in angular scope when grid is initialized
+    columnSavingFlag = false;
```

Comment:
Please update it as discussed and look for other places as well.

Reply:
done